### ISSUE TITLE
Error: Authentication failed

### ISSUE THREAD

**Issue by scottrobertson** [hyperlink/plex-tv-cleanup#26]
Hey

I have this running perfectly on the local machine, but i want to be able to run it from my Raspberry Pi instead. But when i try, i am getting the following:

```
Error: Authentication failed, reason: Invalid status code in authentication response from Plex.tv, expected 201 but got 401
```

The username/password/host are all correct, and they work on the localhost machine.

Any ideas?

**Comment by scottrobertson** [comment]
Oh, this is not even working locally actually.

**Comment by hyperlink** [comment]
Looks like Plex doesn't like the username and password provided. Double-check the values in `.plex-tv-cleanup-config.json`

You can also create your own request in Postman to troubleshoot. Send a POST to `https://plex.tv/users/sign_in.json` with your password and username in Basic Auth. There are some headers that are required such as `X-Plex-Product`, `X-Plex-Version`, and `X-Plex-Client-Identifier` Good luck!

**Comment by scottrobertson** [comment]
I have an idea. I think it may be due to 2FA. You can just append it to the end of the password. Will try that out soon :)

**Comment by scottrobertson** [comment]
Yup, that was it. Appending the 2fa code works. Obviously that won't work when the code expires, so I am not 100% sure the best way to handle this.
